zh-tw/cheatsheet-machine-learning-tips-and-tricks.md

```diff
@@ -0,0 +1,257 @@
+1. **Machine Learning tips and tricks cheatsheet**
+
+&#10230;
+機器學習秘訣和技巧參考手冊
+<br>
+
+2. **Classification metrics**
+
+&#10230;
+分類器的評估指標
+<br>
+
+3. **In a context of a binary classification, here are the main metrics that are important to track in order to assess the performance of the model.**
+
+&#10230;
+在二元分類的問題上，底下是主要用來衡量模型表現的指標
+<br>
+
+4. **Confusion matrix ― The confusion matrix is used to have a more complete picture when assessing the performance of a model. It is defined as follows:**
+
+&#10230;
+混淆矩陣 - 混淆矩陣是用來衡量模型整體表現的指標
+<br>
+
+5. **[Predicted class, Actual class]**
+
+&#10230;
+[預測類別, 真實類別]
```

Comment:
The reason why I mentioned `實際` and `真實` is because your translation on this `actual class` is ˋ真實`, then maybe should modify this one instead of others.